photoview/build.gradle

```diff
@@ -16,24 +17,62 @@ dependencies {
     implementation "androidx.appcompat:appcompat:1.1.0"
 }
 
+def PUBLISH_GROUP_ID = 'com.github.chrisbanes'
+def PUBLISH_ARTIFACT_ID = 'PhotoView'
+def PUBLISH_VERSION = '2.3.0'
```

Comment:
These coordinates might have to be reconsidered as MavenCentral requires verification for publishing with a given group ID